supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesRuntimeAttributes.scala

```diff
@@ -23,13 +26,14 @@ case class TesRuntimeAttributes(continueOnReturnCode: ContinueOnReturnCode,
                                 memory: Option[MemorySize],
                                 disk: Option[MemorySize],
                                 preemptible: Boolean,
+                                localizedSasEnvVar: Option[String],
                                 backendParameters: Map[String, Option[String]])
 
 object TesRuntimeAttributes {
-
   val DockerWorkingDirKey = "dockerWorkingDir"
   val DiskSizeKey = "disk"
   val PreemptibleKey = "preemptible"
+  val LocalizedSasKey = "azureSasEnvironmentVariable"
```

Comment:
Are we 100% sure that custom environment variable names are a requirement? Seems like a possible source of confusion, especially if different WDL authors adopt different conventions.

Reply:
I agree that this is a potential source of confusion. I wanted to go with a `string` rather than a `bool` to give people a means to work around environment variable name collisions and to leave room for getting multiple, different SAS tokens into the environment.